harvester/exceptions.py

```diff
@@ -64,15 +64,15 @@ def __init__(self, msg, harvest_job_id, record_id):
             "harvest_record_id": record_id,  # to-do
         }
 
-        # self.db_interface.add_harvest_error(error_data)
+        self.db_interface.add_harvest_error(error_data)
         self.logger.error(self.msg, exc_info=True)
 
 
 class ValidationException(HarvestNonCriticalException):
     def __init__(self, msg, harvest_job_id, record_id):
         super().__init__(msg, harvest_job_id, record_id)
 
-        # self.db_interface.update_harvest_record(record_id, {"status": "error"})
+        self.db_interface.update_harvest_record(record_id, {"status": "error"})
```

Comment:
don't we want to update the harvest record for any non-critical error? if that's the case then we can move the record update into `HarvestNonCriticalException`

Reply:
Good point! I think so.